tests/fixtures.py

```diff
@@ -22,6 +25,33 @@
 )
 
 
+@pytest.fixture(scope="function", name="current_version_file_path")
+def fixture_current_version_file_path():
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        file_path = os.path.join(
+            PATH_OF_CURRENT_FILE,
+            "h5",
+            "v0.4.2",
+            "MA190190000__2021_01_19_011931__C3__v0.4.2.h5",
+        )
+        new_file_path = migrate_to_latest_version(file_path, tmp_dir)
+        yield new_file_path
+
+
+@pytest.fixture(scope="function", name="trimmed_file_path")
```

Comment:
it looks like there's still a file in the pull request called `tests/h5/v0.4.2/MA190190000__2021_01_19_011931__C3__v0.4.2__trimmed_320_320.h5`.  was that just not deleted after you created this fixture which creates it on the fly (avoiding the need to commit it into the repository)?